rules/autoconsent/cookie-script.json

```diff
@@ -4,6 +4,16 @@
     "prehideSelectors": ["#cookiescript_injected"],
     "detectCmp": [{"exists": "#cookiescript_injected"}],
     "detectPopup": [{"visible": "#cookiescript_injected"}],
-    "optOut": [{"click": "#cookiescript_reject"}],
+    "optOut": [
+        {
+            "if": { "exists": "#cookiescript_reject" },
+            "then": [{" click": "#cookiescript_reject" }],
```

Comment:
```suggestion
            "then": [{"click": "#cookiescript_reject" }],
```

Reply:
I also found a glitch on https://trauer.hna.de/, where the banner doesn't get properly hidden, probably because we're clicking it too fast. Adding a `{ "wait": 100 }` step fixed it.

Reply:
Thanks Sam, good catches!